Readability.js

```diff
@@ -53,6 +53,7 @@ function Readability(doc, options) {
   this._serializer = options.serializer || function(el) {
     return el.innerHTML;
   };
+  this._preferJSONLD = !!options.preferJSONLD || true;
```

Comment:
Temporarily left on `true`, to gauge its effects on the testcases.

Reply:
I think it's fine for it to be an opt-out thing, but then this logic probably needs to be changed, e.g. make the option `disableJSONLD` and update the internal copy accordingly.

Reply:
I've changed the flag to `disableJSONLD`.